### ISSUE TITLE
Inês' iPhone 6 "died" after 2 years for no apparent reason! We *must* complain to Apple!!

### ISSUE THREAD

**Issue by nelsonic** [dwyl/hq#138]
We need to _complain_ to Apple that Inês' iPhone 6 (_which was the furthest possible from "cheap"_) "_died_" after just 2 years of use ... 
@sohilpandya informed us that under UK Consumer Law we can make a _claim_ to get it fixed _for **free**_  within ***6 years*** of purchase. see: http://www.apple.com/uk/legal/statutory-warranty/

Last week I went in and they told me it would be £280 to _repair_ it... 
Obviously the "_geniuses_" at the Apple Store _conveniently_ forgot the "**Statutory rights at no additional cost**" thing... 💸 

We need to get this done _after_ the Christmas _rush_...

**Comment by sohilpandya** [comment]
@nelsonic might be time to do this. 😄

**Comment by markwilliamfirth** [comment]
@iteles did you complain yet?

**Comment by iteles** [comment]
Urgh, no, this isn't high on my list unfortunately

**Comment by markwilliamfirth** [comment]
What's the ROI for complaining? £280?

**Comment by iteles** [comment]
Possibly a new iphone 6? Linked to #152

**Comment by markwilliamfirth** [comment]
If you briefly write out what happened I can write it up properly, submit and handle the complaint? Might be worth it

**Comment by iteles** [comment]
@markwilliamfirth Sure!

Everything was working fine, no issues *at all* or hints that it was about to give up the ghost. 
I was using Google Maps navigation and we were driving along (to the Safe Lives workshop in Bristol), when all of a sudden and without warning, half of the screen went black (half diagonally, as in if a line were to spit the screen from the bottom left to the top right corner, one of those halves went black) and about 2-3 seconds later the whole phone switched went black.

Then I tried switching it on and I got the white apple but it didn't move from there - it froze, wouldn't switch on and wouldn't switch off. So I let it run out of battery completely and it never switched on again after that, even after fully charged.

That's when we went to the apple store.

There are LOTS of un-backed-up photos on there 🤦‍♀️ so I'm loathe to just hand it over without trying to extract some of those through some external service as Apple will just wipe it. Catch 22?

**Comment by nelsonic** [comment]
It's not _just_ about complaining, there are _lots_ of photos on the phone that weren't backed up... 😢 
Complaining is a _necessary **second** step_ (_once the data is recovered_)

**Comment by markwilliamfirth** [comment]
@iteles did you back up your photos with icloud or itunes backup?

**Comment by iteles** [comment]
@markwilliamfirth I use Google Photos NOW to keep them synced but there are 2-3 months of photos on that phone that aren't backed up :(

**Comment by nelsonic** [comment]
@iteles what is the "_Next Action_" for this?

**Comment by markwilliamfirth** [comment]
@nelsonic the next action is to find a way to backup the photos from the dead phone before we give it back. At the moment it's priority 2 so it hasn't ben done yet

**Comment by nelsonic** [comment]
Hi @markwilliamfirth thanks for answering. 👍 
I took the phone to data recovery experts in town a few weeks back and they said the software was corrupted so the data was unreadable. perhaps we could try at another data recovery place ...?
https://www.google.co.uk/search?q=iphone%20data%20recovery
@iteles the longer you leave this the higher the chance of "Data Decay" https://en.wikipedia.org/wiki/Data_degradation#Decay_of_storage_media ⏳

**Comment by markwilliamfirth** [comment]
@iteles are you still interested in us recovering these photos?

**Comment by iteles** [comment]
Very much so, not much luck yet though.

**Comment by markwilliamfirth** [comment]
@iteles will assign this one with you as it makes more sense as these are your personal photos and personal data

**Comment by iteles** [comment]
This is actually a dwyl phone (I hadn't received any money from dwyl yet and couldn't afford one until remuneration was sorted out at the end of the financial year) and a lot of the photos were from dwyl summer, which is why it's in `hq` rather than a personal repo for me, but happy to sort this out of course 👍

**Comment by rub1e** [comment]
Where is the phone now? I shall attend to it

**Comment by iteles** [comment]
It's in my hand as we speak, I've been carrying it around in my work bag for the last 3 weeks and keep forgetting to leave it at the office, will do so tomorrow!

**Comment by rub1e** [comment]
Pop it in the locker and I'll take care of it (as discussed, the key is actually where it always was, in the box)

**Comment by rub1e** [comment]
I've got a place that's well reviewed called `iSmash`; I'll take it in next time I'm in town so we save on the postage 🏣

**Comment by iteles** [comment]
@rub1e I handed my phone to @sofer who kindly agreed to bring it in to the office. Did he hand it to you?

**Comment by rub1e** [comment]
@iteles Yeah all good, it's in my bag. Depending on where I watch the footie, I may even hand it in today, but currently unclear

**Comment by rub1e** [comment]
Just had a call from the technician; he says that they ran the diagnostic and the phone isn't in a good place - even with a new battery, it won't connect to a laptop, which means it's a `logic board` issue

So the only solution is to get the whole phone fixed, as they can't do any data recovery without the phone working in the first place, and that will take 2-3 weeks as they have to send it to their central hub

BUT the good news is that the logic board fix comes in at £20 cheaper than the data recovery AND we won't actually need the data recovery because that will be sorted with the logic board fix

tl;dr - £99.99 gets you your phone fixed; it'll take up to three weeks, but you won't need the data recovery

How does that sound @iteles ?

**Comment by nelsonic** [comment]
Hi @rub1e _thank you_ for investigating this! 🔍 👍 

a "logic board fix" will mean that the "_old_" Logic Board is _discarded_. i.e. data will be _lost_.
Please _confirm_ (_by email i.e. in writing_) with the person/people offering to "fix" the phone 
that the memory chip on the phone will not be substituted, otherwise this is a _pointless_ "fix".

To be clear: we don't want the phone to be "fixed" without ***recovering*** the photos on the phone.
The photos on the device include _deceased_ relatives, i.e. they are _priceless_. 💔 😢 
If someone offers to recover ***all*** the data we are willing to pay £1k (_the price of a `new` iPhone XS!_) 💰 
if they cannot _guarantee_ that the data will _not_ be _lost_ we aren't _minimally_ interested in a "cheaper fix".

Thanks! ✨

**Comment by rub1e** [comment]
> the memory chip on the phone will not be substituted

Yes that was my understanding, but absolutely will check and get that in writing

> we don't want the phone to be "fixed" without recovering the photos on the phone

100%

> if they cannot guarantee that the data will not be lost we aren't minimally interested in a "cheaper fix".

If they can't guarantee the data, I'll go elsewhere

**Comment by iteles** [comment]
@rub1e Thanks for this. If we can get it in writing, that would be ideal. Thanks again for looking into it!

**Comment by rub1e** [comment]
How's this for a guarantee? Let me know and I'll get them to start

![screenshot 2018-10-24 at 14 18 20](https://user-images.githubusercontent.com/11595920/47433219-e2ce3b00-d797-11e8-8463-d8916fd71ac1.png)

**Comment by rub1e** [comment]
OK, this has been approved, it's going to the next level repair place, and I'll update this when they've given me a timeline

(And the guy on the phone reassured me that they would call me if they had any worries about affecting the data before they start)

And we don't have to pay until we collect the repaired phone.

**Comment by rub1e** [comment]
I emailed to ask why we hadn't heard anything and oh! what a coincidence - I got a note saying I can come pick the phone up on the 6th of December at 8am. :man_facepalming: 

I'm sure I'll pass through Kx in the next few days so I'll pick it up and send it on to Braga

**Comment by iteles** [comment]
@rub1e Did they say whether they'd been able to retrieve any data?

**Comment by nelsonic** [comment]
@rub1e thanks for persisting with this. 🥇 
I don't know if it's "_urgent_" to send the phone to Braga ... (_given how long we've already waited..._)
@iteles could it wait for when someone comes to visit @home in the next couple of months?

**Comment by rub1e** [comment]
Cool! It also occurs to me that I could bring it in to the office to charge and maybe connect it to someone's iCloud account - or even just download all of the files onto the hard drive and upload them to Google Drive from there? (I couldn't do this at home as I have no access to Apple Tech, not even cables)

**Comment by rub1e** [comment]
@Cleop is making a note to bring in the cables so we can do this ☝️

**Comment by nelsonic** [comment]
@rub1e again no rush. as long as the data is "safe" on the device and the device is in a safe place ...

**Comment by iteles** [comment]
This was sorted successfully in the first quarter of this year. Thanks @rub1e 🌟
